src/main/java/io/javalin/Javalin.java

```diff
@@ -54,6 +50,7 @@
     private LogLevel logLevel = LogLevel.OFF;
     private String defaultContentType = "text/plain";
     private String defaultCharacterEncoding = StandardCharsets.UTF_8.name();
+    private long maxRequestCacheBodySize = 8 * 1024;
```

Comment:
Why was this number chosen?

Reply:
Was taken accidentally from some Java defaults. Changed to Long.MAX_VALUE.